dev/Breadcrumb/BreadcrumbBar.cpp

```diff
@@ -240,7 +240,8 @@ void BreadcrumbBar::OnElementPreparedEvent(const winrt::ItemsRepeater&, const wi
                 m_ellipsisBreadcrumbBarItem.set(item);
                 UpdateEllipsisBreadcrumbBarItemDropDownItemTemplate();
 
-                winrt::AutomationProperties::SetName(item, ResourceAccessor::GetLocalizedStringResource(SR_AutomationNameEllipsisBreadcrumbBarItem));
+                winrt::AutomationProperties::SetName(args.Element(), s_ellipsisBreadcrumbItemAutomationName);
```

Comment:
Why make this change?  A large majority of our winrt::AutomationProperties::SetName(...) calls use localized strings.

Is SR_AutomationNameEllipsisBreadcrumbBarItem still used?

Reply:
that change was a mistake, I changed it in bundle with the enumerated name for the regular breadcrumb item so I reverted the change